fem/gslib.cpp

```diff
@@ -917,15 +917,15 @@ void FindPointsGSLIB::InterpolateH1(const GridFunction &field_in,
 
    const int ncomp      = field_in.FESpace()->GetVDim(),
              points_fld = field_in.Size() / ncomp,
-             points_cnt = gsl_code.Size();
+             points_cnt_ = gsl_code.Size();
```

Comment:
Actually at this point, we should not re-define `points_cnt` since it is set in `FindPointsGSLIB::FindPoints`. Instead we should verify the size of `gsl_code` (and probably other arrays like `gsl_proc`) is `points_cnt`.
Maybe we can do the check only in debug mode.

Reply:
done in [f60272a](https://github.com/mfem/mfem/pull/3750/commits/f60272ab0bb29583de9e4c1479f5bb1a8e2dbd42)